lib/ut-coverage/test_bplib_base_setup.c

```diff
@@ -35,12 +35,31 @@ void UT_lib_sizet_Handler(void *UserObj, UT_EntryKey_t FuncKey, const UT_StubCon
     UT_Stub_SetReturnValue(FuncKey, retval);
 }
 
+void UT_lib_sizet_1_Handler(void *UserObj, UT_EntryKey_t FuncKey, const UT_StubContext_t *Context)
+{
+    size_t    retval = 1;
+    UT_Stub_SetReturnValue(FuncKey, retval);
+}
+
```

Comment:
We should not need to add a separate handler just to return 1 instead of 0.  For example see the `UT_DefaultHandler_bplib_os_get_dtntime_ms` function, this simply extends the 32 bit return value to 64 bits, such that the regular return value functions can be used with it.

My recommendation would be to make a clone of the `UT_DefaultHandler_bplib_os_get_dtntime_ms` function for now.  Eventually this logic should be built into UT assert as a default so we don't need to make a handler at all, but until that happens, this should suffice.